assets/bower.json

```diff
@@ -43,7 +44,8 @@
   },
   "appPath": "app",
   "resolutions": {
-    "lodash": "3.10.1"
+    "lodash": "3.10.1",
+    "fontawesome": "4.5.0"
```

Comment:
@rhamilto heads up, we need a newer version of fontawesome than is included with patternfly, would suggest we push them to upgrade unless there is a reason not to...  but for now we need to override it so we can get this in to 1.2